opnode/test/setup.go

```diff
@@ -40,14 +43,30 @@ func deriveAccount(w accounts.Wallet, path string) accounts.Account {
 	return account
 }
 
+type L2OOContractConfig struct {
+	SubmissionFrequency   *big.Int
+	L2BlockTime           *big.Int
+	GenesisL2Output       [32]byte
+	HistoricalTotalBlocks *big.Int
+}
+
+type DepositContractConfig struct {
+	L2Oracle           common.Address
+	FinalizationPeriod *big.Int
+}
```

Comment:
Would prefer `uint64` in config structs here. And should these have a `Check()` method?

Reply:
These are all internal and they end up as `big.Ints` so I don't see a big reason to hide that fact.